### DISCUSSION TITLE
微软实践开源的5条真知灼见;如何建立和培养一个伟大的开发者共同体；等开源之道一周评论 2021 08 08 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#64] [Announcements]
# 微软实践开源的5条真知灼见;如何建立和培养一个伟大的开发者共同体；等开源之道一周评论 2021 08 08 - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2021/%E5%BC%80%E6%BA%90%E4%B9%8B%E9%81%93%E6%AF%8F%E5%91%A8%E8%AF%84%E8%AE%BA-2021-08-01/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这种周刊形式, 推荐者的断言很重要,
不过, 推荐尝试 DUKU 三志气:

+ 有趣
+ 有用
+ 有种

最关键是最后一条, 不迎合不为反对而反对, 也不为出圈而喧嚣...

有独立人格和思考的成果...

否则, 相似资料, 获得相似反馈太乏味儿...

从标题来看, M$ 无论以后多么开源, 都不可能洗去对 FLOSS 世界的攻击历史...
所以, 其应激式反应根本无法令人信服...
